### PR TITLE
[enterprise-4.8] OSS-Docs-32: Removed instances of OCM

### PR DESCRIPTION
**Pull request by EricPonvelle**
This is the manual cherry pick of #41055

### CODE REVIEW

File: modules/add-user.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
```
Comment (kalexand-rh) [inline modules/add-user.adoc:1]: Do you need this file?

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 2]
```diff
@@ -19,15 +19,26 @@ The following Technology Preview features are enabled by this feature set:
 ** CSI automatic migration. Enables the automatic migration of supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Available as a Technology Preview for:
 *** Amazon Web Services (AWS) Elastic Block Storage (EBS)
 *** OpenStack Cinder
+*** Azure Disk
```
Comment (kalexand-rh) [inline modules/nodes-cluster-enabling-features-about.adoc:22]: Is this right?

File: modules/telemetry-consequences-of-disabling-telemetry.adoc [hunk 3]
```diff
@@ -1,21 +0,0 @@
-// Module included in the following assemblies:
```
Comment (kalexand-rh) [inline modules/telemetry-consequences-of-disabling-telemetry.adoc:1]: Did you mean to remove this?

**Review by kalexand-rh — COMMENTED** [review]
This is looking very close!

File: modules/nodes-cluster-enabling-features-about.adoc (hunk 2 above)
Comment (EricPonvelle) [reply modules/nodes-cluster-enabling-features-about.adoc:22]: That's how it's listed in `main` if I am reading that file right

Comment (kalexand-rh) [reply modules/nodes-cluster-enabling-features-about.adoc:22]: Right, but do you know if it's valid in 4.8? I suggest not making this branch match main here.

**Comment by skrthomas** [comment]
lgtm

**Pull request merged**